<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TodoMainWindow</class>
 <widget class="QMainWindow" name="TodoMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Todo List</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="lineEdit">
      <property name="placeholderText">
       <string>Enter a task...</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="todoAddButton">
        <property name="text">
         <string>Add Task</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="todoDeleteButton">
        <property name="text">
         <string>Delete Task</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Priority</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout2">
       <item>
        <widget class="QRadioButton" name="radioButtonHigh">
         <property name="text">
          <string>High</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButtonMedium">
         <property name="text">
          <string>Medium</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButtonLow">
         <property name="text">
          <string>Low</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>

        <item>
            <layout class="QHBoxLayout" name="horizontalLayout1">
                <item>
                <widget class="QPushButton" name="sortAlphabeticalButton">
                    <property name="text">
                    <string>Sort Alphabetically</string>
                    </property>
                </widget>
                </item>
                <item>
                <widget class="QPushButton" name="sortPriorityButton">
                    <property name="text">
                    <string>Sort by Priority</string>
                    </property>
                </widget>
                </item>
            </layout>
        </item>
    
    <item>
     <widget class="QListWidget" name="listWidget"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
